### DISCUSSION TITLE
Problem with Elasticsearch Snapshots

### DISCUSSION THREAD

**Discussion by OrSo1201** [Security-Onion-Solutions/securityonion#9182] [Unsupported Versions]
Hi,
We are trying to use the Elasticsearch Snapshot feature.
The snapshot are created at the salt server but not in the Elastic web platform.
How can we figure out this problem?
Thank You

**Comment by dougburks** [comment]
Have you checked the documentation at https://docs.securityonion.net/en/2.3/backups.html#elasticsearch?
